frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -641,6 +686,14 @@ export class TileViewPage extends React.Component {
     this.setState({ filterGroupsShowAll: updatedShow });
   }
 
+  onDropdownChange(value) {
+    if (value === 'None') {
+      this.onFilterChange('groupBy', null, '');
+    } else {
+      this.onFilterChange('groupBy', null, value);
+    }
```

Comment:
Why is it not simply `this.onFilterChange('groupBy', null, value);`

Although grouping is not a filter, so you shouldn't be using `onFilterChange`.

Reply:
Not using `onFilterChange` anymore